### ISSUE TITLE
Will there be a new version?? Kindly DEV teams please update us

### ISSUE THREAD

**Issue by fisforfaheem** [invertase/zapp.run#191]
we really want this to grow please bring it back, this is 10x better than google project idx

**Comment by Ehesp** [comment]
Hey, yes there will be and we're working on it. 

No exact ETA yet, but expect Q1 25.

**Comment by fisforfaheem** [comment]
kindly atleast say COMING SOON ON YOUR WEBSIT :(

**Comment by mobiledevenv** [comment]
> Hey, yes there will be and we're working on it. 
> 
> No exact ETA yet, but expect Q1 25.

Q1 January?It's March now Can't it just be open source?I don't care if there are errors in the source code,I can update and fix it myself

**Comment by mobiledevenv** [comment]
> Hey, yes there will be and we're working on it. 
> 
> No exact ETA yet, but expect Q1 25.

Q1 January?It's March now Can't it just be open source?I don't care if there are errors in the source code,I can update and fix it myself

**Comment by mobiledevenv** [comment]
> Hey, yes there will be and we're working on it. 
> 
> No exact ETA yet, but expect Q1 25.

Q1 January?It's March now Can't it just be open source?I don't care if there are errors in the source code,I can update and fix it myself

**Comment by Ehesp** [comment]
We had to change some priorities around for reasons (commercial & resourcing) so things have been delayed. Apologies, I know it's not ideal, but we're doing our best with limited capacity to move things along.

**Comment by fisforfaheem** [comment]
yes all we ask is to atleast use AI t update your website and ateleast write ocmign soon there

**Comment by Ehesp** [comment]
Hey All - we've made some really good progress on things, we'll have an update soon :)

**Comment by fisforfaheem** [comment]
Hi Ehesp,

Thank you for the update on the progress made. We appreciate your efforts
and look forward to the upcoming update.

Best regards,

**Comment by MengLizhi** [comment]
As of 2025-08-26 15:12:37, only Flutter 3.7.3 version can still be used. Has this project been abandoned?

**Comment by fisforfaheem** [comment]
its 2026 now :(
